docker/sandbox/sandbox-sus2.yaml

```diff
@@ -64,14 +64,7 @@ metadata:
     traefik.frontend.priority: "1"
 spec:
   rules:
-    - host: "mediawiki.prod.sjc.k8s.wikia.net"
-      http:
-        paths:
-          - path: /
-            backend:
-              serviceName: mediawiki-sandbox
-              servicePort: 80
-    - host: "*.wikia.com"
+    - host: "*.sandbox-sus2.wikia.com"
```

Comment:
This will likely require adjusting `traefik.frontend.priority`, since `*.wikia.com` covers `*.sandbox-sus2.wikia.com`